### PR TITLE
Jwt

### LINKED ISSUE #10: randomize worker-id
Radomize worker ids, so worker-service requests cannot be spammed to lower or higher worker id´s

### PR DESCRIPTION
**Pull request by LeanderK**
implemented JsonWebToken functionality, Fixes #10

### CODE REVIEW

**Comment by LeanderK** [comment]
depends on the worker-ui

**Comment by LeanderK** [comment]
@phimonk you can merge when the worker-ui is ready

File: src/test/java/edu/kit/ipd/crowdcontrol/workerservice/JWTHelperTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    public void testGenerateJWT() throws Exception {
+        String jwt = jwtHelper.generateJWT(13);
+        assertEquals(jwt, "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiIxMyJ9.8hs07EsZJLZAjU8moUWJqPQjT3Yqw8ACNLfow7VsEqo");
+    }
+
+    @Test
+    public void testGetWorkerID() throws Exception {
+        String jwt = jwtHelper.generateJWT(13);
+        int workerID = jwtHelper.getWorkerID(jwt);
+        System.out.println(workerID);
```
Comment (phimonk) [inline src/test/java/edu/kit/ipd/crowdcontrol/workerservice/JWTHelperTest.java]: maybe assert worker id = 13?

Comment (LeanderK) [inline src/test/java/edu/kit/ipd/crowdcontrol/workerservice/JWTHelperTest.java]: uh, yeah i reverted that to test it :D

**Pull request merged**